specification/guestconfiguration/GuestConfiguration.Management/legacy.tsp

```diff
@@ -0,0 +1,314 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.Legacy {
+  interface CustomLegacyOperations<
```

Comment:
Whats our plan - are we going to file a ticket on the RP to release a new version addressing the legacyness, or are we going to keep it legacy forever?

Reply:
No. Our plan is to remove such custom legacy operations.